### ISSUE TITLE
use a better compression

### ISSUE THREAD

**Issue by razr** [ros-realtime/ros-realtime-rpi4-image#57]
I see that the image is compressed by the `pigz` from the original 3.9G to 1.5G, however it looks like that @carlossvg produces a release `xz` compression which is only 932 MB

```bash
Before compression:
total 3.9G
-rw-r--r-- 1 runner docker 4.0G Sep 27 13:43 ubuntu-22.04.1-rt-ros2-arm64+raspi.img
+ cd out
+ echo 'After compression:'
+ pigz ubuntu-22.04.1-rt-ros2-arm64+raspi.img
After compression:
+ ls -lh .
total 1.5G
```

**Comment by shuhaowu** [comment]
Is that in CI? When I updated the release instructions, I opted to use xz as it produces a smaller image: https://github.com/ros-realtime/ros-realtime-rpi4-image/blob/master/docs/ReleaseImage.md

**Comment by razr** [comment]
yes, it is in the GitHub workflow. I have built `jammy` https://github.com/razr/ros-realtime-rpi4-image/actions/runs/3135956272/jobs/5092263112
Right, if it would be nice to use `xz`, we can create an artifact and download it then.

**Comment by shuhaowu** [comment]
Ah good point. We can switch the CI to using xz, but we also need to switch it to manual activation if that's the case (instead of automatically running on master push), because my experience running `xz` is that it is very slow and may consume a large amount of free CI time Github gives out for a particular account. 

These should be simple changes but I don't have time to do that right now. Do you have time to create a PR and we can merge it?

**Comment by razr** [comment]
I agree, I'm also using `workflow_dispatch` for the manual start in my other projects. I believe @LanderU wants to provide a patch.

**Comment by carlossvg** [comment]
Separate workflow in 2 steps:
- build
- compress

**Comment by shuhaowu** [comment]
I've ended up using zstd. It strikes a balance between compression rate and compression speed.
